frontend/src/services/web3-provider/types.ts

```diff
@@ -37,3 +38,8 @@ export type Eip1193Provider = ExternalProvider & {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   on(eventName: string, listener: (...args: any[]) => void): void;
 };
+
+export type DetectedEthereumProvider =
+  | ExternalProvider & {
```

Comment:
This was enforced by prettier for some reasons and i don't get why.. 
Since it doesn't affect the type i didn't want to go deeper and waste time on figuring out how prettier does its thing under the hood.